File: sqlInsert.py
import MySQLdb

db = MySQLdb.connect(host="localhost",user="user",passwd="password",db="school"

# creating cursor
cur = db.cursor(MySQLdb.cursors.DictCursor);

name= 'Jason'
age= 17
gradeLevel= 11

# create table as per requirement
sql = "INSERT INTO students (name,age,gradeLevel) VALUES ('Jason',17,11)"
cur.execute(sql)

# disconnecting from server
cur.close()
db.close()

File: sqlInsert.php
<?php
 
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";
 
// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->) {
   die("Connection failed: " . $conn->connect_error);
}
 
$name = 'Landon';
$age = '9';
$gradeLevel = '17';
 
$sql = "INSERT INTO students (name, age, gradeLevel)
VALUES ('$name','$age', '$gradeLevel')";
 
if ($conn->query($sql) === TRUE) {
   echo "New record created successfully";
}
else {
   echo "Error: " . $sql . "<br>" . $conn->error;
}
 
$conn->close();
?>

